Ginger/GingerCoreCommon/VariablesLib/VariableSelectionList.cs

```diff
@@ -55,6 +58,24 @@ public string OptionalValues
 
         public string SelectedValue { set { Value = value; OnPropertyChanged(nameof(SelectedValue)); } get { return Value; } }
 
+        private bool mRandomOrder;
+        [IsSerializedForLocalRepository]
+        public bool RandomOrder
+        {
+            set
+            {
+                mRandomOrder = value;
+                OnPropertyChanged(nameof(this.RandomOrder));
+            }
+            get
+            {
+                return mRandomOrder;
+            }
+        }
+
+        [IsSerializedForLocalRepository]
```

Comment:
do we really need the "CurrentValueIndex" to be saved to XML or at all this field? why not return the next value after the current selected Value?